yarn.lock

```diff
@@ -3,55 +3,45 @@
 
 
 "@1inch/permit-signed-approvals-utils@^1.4.10":
-  version "1.4.10"
-  resolved "https://registry.npmjs.org/@1inch/permit-signed-approvals-utils/-/permit-signed-approvals-utils-1.4.10.tgz"
-  integrity sha512-wWRO5h0a2rzQgIbbNrq1wtGhmDKsLq7DwuT3qfuXCrt6mfkEAem9EJnT8Q09mCGOY/mLUQhEorKbbcHOEGi/7A==
+  version "1.5.2"
+  resolved "https://registry.yarnpkg.com/@1inch/permit-signed-approvals-utils/-/permit-signed-approvals-utils-1.5.2.tgz#3f299e0197f8cab27740cfe7a741f6d1120ce010"
+  integrity sha512-M3l1YdSSzdsRTusiPkcfgCQ25ZLJeC41LUWDOrunGMpl4F/D058FtXk0e5lxMJ7Wi+CmIKxmm7KVUFtIiEi1dw==
   dependencies:
-    "@metamask/eth-sig-util" "^4.0.1"
-    web3 "^1.5.1"
-    web3-utils "^1.5.1"
+    ethers "^6.13.1"
```

Comment:
I'm not sure we can merge this as it's upgrading this dep to ethers v6, and the way we use it might depend on web3js